md/product-versioning.md

```diff
@@ -0,0 +1,51 @@
+# Bonita Versioning
+Discover how Bonita manages its product and artifacts versions and how it can impact your projects
+
+## Product Versions
+
+Bonita follows the usual semantic versioning X.Y.z:
+* X is the major version number. It changes when non-backward compatible changes are implemented.
```

Comment:
Is it always true? Can't we release a 8.0 without compatibility break?

Reply:
Anything is possible. I rather be restrictive here.